packages/react-core/src/components/Banner/examples/BannerBasic.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Banner, Flex } from '@patternfly/react-core';
+
+export const BannerBasic: React.FunctionComponent = () => (
+  <Flex direction={{ default: 'column' }}>
+    <Banner>Default banner</Banner>
```

Comment:
Usually our examples match the core examples.  For that reason, I would add back the `<br />`

Reply:
Done. And updated other recent instances where I was using flexbox instead of breaks and empty spaces as well.